### ISSUE TITLE
Move mailing list signup?

### ISSUE THREAD

**Issue by ikarth** [falonofthetower/howling-hematite#112]
The mailing list sign up checkbox is really cluttering up the donation form. Do we want to move it to a button on the Thank You page? Which is a logical place to put it, and less likely to scare people off. What would that take? The email would have to be carried over or re-entered at a minimum...

**Comment by ikarth** [comment]
This is probably going to need some back end changes if we decide to do it.

**Comment by falonofthetower** [comment]
Yes, yes it will. Check with mother and confirm we want to do this and I'll build it.
